extensions/labgraph_viz/setup.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from setuptools import find_packages, setup
+
+
+setup(
+    name="labgraph_viz",
+    version="1.0.0",
+    description="Some useful visualizations for labgraph",
+    packages=find_packages(),
+    python_requires=">=3.6, <3.7",
+    install_requires=[
+        "labgraph==1.0.0",
```

Comment:
I think this should list some additional deps:
```
dataclasses
matplotlib
numpy
PyQt5-sip
pyqtgraph
```

Reply:
Libraries unique to LabGraph-Viz could be considered to be added, e.g.:
PyQt5-sip
pyqtgraph

Reply:
Yes, __init__.py needs a copyright line, thanks for spotting it! Reference: https://github.com/facebookresearch/labgraph/blob/master/labgraph/graphs/tests/__init__.py

Reply:
Added :)